charts/internal/gvisor/templates/rbac.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: gvisor
+  namespace: kube-system
+spec:
+  privileged: true
+  volumes:
+    - 'configMap'
+    - 'secret'
+    - 'hostPath'
+  hostNetwork: false
+  hostIPC: true
+  hostPID: true
+  allowedHostPaths:
+    - pathPrefix: /
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    rule: 'RunAsAny'
+  supplementalGroups:
+    rule: 'RunAsAny'
+  fsGroup:
+    rule: 'RunAsAny'
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can you move every resource into its own file? I.e., one file for PSP, one for ClusterRole, etc.